birdhouse/env.local.example

```diff
@@ -381,33 +381,35 @@ export GEOSERVER_ADMIN_PASSWORD="${__DEFAULT__GEOSERVER_ADMIN_PASSWORD}"
 #export JUPYTERHUB_AUTHENTICATOR_REFRESH_AGE=60
 
 # Resource limits for JupyterLab containers. Resource limits can be set per Magpie user or group.
-# The value for this variable is a whitespace delimited string. Each section is delimited by colons (:) 
-# where the first element is either `group` or `user` and the second element is the name of the user or group
-# to apply the limits to. The rest are resource limits of the form `limit=amount`. For example:
-#
-#  export JUPYTERHUB_RESOURCE_LIMITS="
-#     user:user1:mem_limit=30G
-#     group:group1:mem_limit=10G:cpu_limit=1
-#     group:group2:cpu_limit=3:gpu_ids=0,3,4
-#     user:user2:gpu_ids=1,2,3:gpu_count=2
-#  "
-#
-#  Supported limits are: `mem_limit`, `cpu_limit`, `gpu_count`, `gpu_ids`. See the Jupyterhub Dockerspawner documentation
+# The value for this variable is a yaml or JSON array of mappings with the following keys: "type" (either "user" 
+# or "group"), "name" (the name of the group or user to apply the limits to) and "limits" (see below). For example:
+# export JUPYTERHUB_RESOURCE_LIMITS='
+#  [
+#    {"type": "user", "name": "user1", "limits": {"mem_limit": "30G"}},
+#    {"type": "group", "name": "group1", "limits": {"mem_limit": "10G", "cpu_limit": 1}},
+#    {"type": "group", "name": "group2", "limits": {"cpu_limit": 3, "gpu_ids": [0, 3, 4]}},
+#    {"type": "user", "name": "user2", "limits": {"gpu_ids": [1, 2, 3], "gpu_count": 2}}
+#  ]
+#'
+#  Supported limits are: "mem_limit", "cpu_limit", "gpu_count", "gpu_ids". 
+#  For a JSON schema describing the structure of this JSON array see 
+#  birdhouse/components/jupyterhub/resource-limit.schema.json
+#  See the Jupyterhub Dockerspawner documentation
 #  for details and supported values for mem_limit and cpu_limit.
 #   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.cpu_limit
 #   - https://jupyterhub-dockerspawner.readthedocs.io/en/latest/api/index.html#dockerspawner.DockerSpawner.mem_limit
-#  gpu_ids are a comma separated list of the GPU uuids or zero based indexes available on the host that you want to make available 
+#  gpu_ids are an array of the GPU uuids or zero based indexes of the GPUs that you want to make available 
```

Comment:
Can we mix gpu uuid with gpu indexes?  Or it's only one type or the other?  If only one or the other, please add a note.